.github/workflows/codeql.yml

```diff
@@ -29,5 +29,10 @@ jobs:
           languages: go
           queries: security-and-quality
 
+      - name: Setup go
+        uses: actions/setup-go@v5
+        with:
+          go-version: '1.22'
```

Comment:
Shouldn't this be reading the version from the `go.mod` instead?

Reply:
I agree, but I will follow up with a PR to make it so. This PR doing the bump and another doing the refactoring of our workflows.